### ISSUE TITLE
Wrong address using own WIF private key

### ISSUE THREAD

**Issue by ahparhizgar** [walletgeneratornet/WalletGenerator.net#266]
in the second tab (paper wallet) if you past your own private key after clicking "Apply" button it shows a wrong address. and every time a new address!! (for a specific private key)
**It will waste people funds.**

**Comment by epsi1on** [comment]
Seems website is different than github repo.
github code works fine but online version not OK!!!

**Comment by dabba39** [comment]
yep, true.
Solution: download the zip and open the index.html with your webbrowser, it will work out of the box.

**Comment by epsi1on** [comment]
Anyways be **CAREFUL**. It is possible to send BTC to a wallet address which is not belong to you and probably is someones else's wallet (probably coder's wallet!!!)

This is called SCAM or THIEF or STEALING BTC which WalletGenerator.net is doing...

**Comment by kelalaka153** [comment]
Yes, the site has problems, it randomly shows the correct public-private key on different view clicks. I was lucky to get the correct one on the first click on my first try. Then it started to change. [Here my experiments](https://crypto.stackexchange.com/a/90214/18298)
